web-common/src/features/dashboards/pivot/pivot-merge-filters.spec.ts

```diff
@@ -24,6 +28,34 @@ describe("mergeFilters", () => {
     ).toEqual(`publisher IN ('Facebook') AND publisher NIN ('Google',null)`);
   });
 
+  it("merge like and subquery filters", () => {
+    expect(
+      convertExpressionToFilterParam(
+        mergeFilters(
+          createAndExpression([
+            createInExpression("publisher", ["Facebook", "Yahoo"]),
+            createLikeExpression("publisher", "%oo%"),
+          ]),
+          createAndExpression([
+            createInExpression("publisher", ["Facebook", "Microsoft"]),
+            createSubQueryExpression(
+              "publisher",
+              ["impressions"],
+              createBinaryExpression(
+                "impressions",
+                V1Operation.OPERATION_GT,
+                2,
+              ),
+            ),
+          ]),
+        )!,
+      ),
+    ).toEqual(
+      // Support for LIKE is in another PR. Enable this before that PR is merged.
```

Comment:
Is this comment still relevant?

Reply:
Ya this PR adds support for `like` https://github.com/rilldata/rill/pull/6773